nucliadb/nucliadb/ingest/chitchat.py

```diff
@@ -112,7 +112,7 @@ async def socket_reader(
                                 listen_addr=x["address"],
                                 node_type=x["type"],
                                 is_self=x["is_self"],
-                                load_score=x.get("load_score", 0.0),
+                                load_score=float(x.get("load_score", "0")),
```

Comment:
Could we try except this operation ?

Reply:
Yes sure. In case of failure, log it and default load score to zero ?

Reply:
yes